tests/unit/lms/services/assignment_test.py

```diff
@@ -425,13 +425,14 @@ def test_get_assignments_returns_all_assignments_for_instructor(
         )
         db_session.flush()
 
-        assert set(
-            db_session.scalars(
+        assert {
+            assignment.id
+            for assignment in db_session.scalars(
                 svc.get_assignments(
                     instructor_h_userid=instructor_in_assignment.h_userid
                 )
             ).all()
-        ) == {assignment, assignment_not_launched_by_instructor}
+        } == {assignment.id, assignment_not_launched_by_instructor.id}
```

Comment:
I'm not entirely sure why this fails now and not before.

Anyway a set of assignments is not a great idea, gone for a set of IDs.